src/main/java/duke/Task.java

```diff
@@ -0,0 +1,61 @@
+package duke;
+
+/***
+ *  The superclass of all types of Tasks created by the user
+ */
+public class Task {
+    private String details;
```

Comment:
Perhaps you would like to make Task an abstract class? Especially since you would not need to instantiate this class directly. 🤔